README.md

```diff
@@ -12,14 +12,16 @@ and a locally unique identifier (LUI)" [Wimalaratne2018]. For example, for prote
 represent the PDB structure [2gc4](https://bioregistry.io/pdb:2gc4) as *pdb:2gc4*.
 
 When the prefixes are defined by community standards, then a compact identifier can be resolved.
-There currently two main providers of prefix files [Wimalaratne2018], Identifiers.org [BernalLlinares2021],
-and Bioregistry [Hoyt2022]. There match three existing *resolvers* that will take a compact
+The Bioregistry indexes more than 20 widely used registries of prefixes and their associated metadata
+on [https://bioregistry.io/related](https://bioregistry.io/related). Three notable registries are
```

Comment:
how do you want to position these? Name-to-Thing is only a resolver and doesn't have a nice UI, so it's therefore not a good tool to point people at who need to find new prefixes

Reply:
For historical reasons and to demo the international character, I like to keep Name-to-Things.